docs/python_sdk_v2.0.md

```diff
@@ -133,7 +133,6 @@ cv.imwrite('beatles_output.jpg', im)
 import os
 from xailient import roi_bbox
 import skimage.io
-import cv2 as cv
```

Comment:
seeing the full code I see we are doing
```py
cv.rectangle(im, pt1, pt2, (0, 255, 0))
```
so need this import in here (I'm really sorry for the many goes and back, didn't saw the full context before)